noah/src/main/scala/edu/uci/ics/cloudberry/noah/kafka/ConsumerKafka.java

```diff
@@ -0,0 +1,72 @@
+package edu.uci.ics.cloudberry.noah.kafka;
+
+
+import edu.uci.ics.cloudberry.noah.feed.AsterixHttpRequest;
+import edu.uci.ics.cloudberry.noah.feed.Config;
+import edu.uci.ics.cloudberry.noah.feed.TagTweetGeotagNotRequired;
+import edu.uci.ics.cloudberry.noah.feed.TwitterFeedStreamDriver;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.kohsuke.args4j.CmdLineException;
+import twitter4j.TwitterException;
+
+import java.util.Arrays;
+import java.util.Properties;
+
+public class ConsumerKafka {
+
+    private Properties getProperties(String server, String groupId) {
```

Comment:
This is better to be stored in a config file than hard coded. @aishwaryakapse  knows how to do it.